backend/src/emails/sendEmailTranslated.ts

```diff
@@ -0,0 +1,82 @@
+import { backendLogger as logger } from '@/server/logger'
```

Comment:
this functionality should be replacing the existing sendMail to have backward comptibility.
-> All untranslated/not html emails should be send the old way
-> All already properly defined emails should be sent the new way

Reply:
That's what I have in mind too! 🤗